SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1370,13 +1370,22 @@ void USpatialActorChannel::ClientProcessOwnershipChange(bool bNewNetOwned)
 	SCOPE_CYCLE_COUNTER(STAT_ClientProcessOwnershipChange);
 	if (bNewNetOwned != bNetOwned)
 	{
```

Comment:
It looks like it will call the new functions with the old value without this?
```suggestion
	{
		bNetOwned = bNewNetOwned;
```